### ISSUE TITLE
Multi-tenancy issue with Acquiretokensilent MSAL js

### ISSUE THREAD

**Issue by athreyee** [AzureAD/microsoft-authentication-library-for-js#797]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Browser:
- [x ] Chrome version XX
- [x ] Firefox version XX
- [x ] IE version XX
- [x ] Edge version XX
- [x ] Safari version XX

## Library version- 1.0.1 MSAL.js

<pre><code>
Library version: 1.0.1

## Current behavior

When user are trying to login with different domain accounts, Acquiretokensilent method is failing with AADSTS65005: The application needs access to a service that your organization test has not subscribed to. We have already added our client id in the manifest of our Web API under "Known Applications" tag but still the issue persists

## Expected behavior

user should be able to login and get the access token

## Minimal reproduction of the problem with instructions

1. User logins with user2@uumtest1.net and UserAgentApplications is initialised with authority https://login.microsoftonline.com/common/
2. While trying to get access token acquiretokensilent fails with exception
AADSTS65005: The application needs access to a service that your organization test has not subscribed to

3. The network calls show that the url getting it is incorrect, instead of common , tenantid is getting pulled
https://login.microsoftonline.com/db776ee0-145c-47b0-a1b9-8a4da2dad75f/oauth2/v2.0/authorize?response_type=token&scope=https%3A%2F%2Fmicrosoft.onmicrosoft.com%2Fa9404161-811a-480e-a86c-082d92b9f4a2%2Fuser_impersonation%20openid%20profile&client_id=db7116e3-dbeb-4336-9525-

</code></pre>

**Comment by sameerag** [comment]
@athreyee Can you please share the correlation-id and timestamp for this issue? I forwarded this to the app registration team to confirm if your settings look fine and this info would help understand the AAD error better.

**Comment by athreyee** [comment]
The request url getting hit is below while calling acquiretokensilent for the app id db7116e3-dbeb-4336-9525-07bd8dc6fede

x-ms-request-id: 0a051d3a-0c29-4696-a7ef-6b2e6d122400




https://login.microsoftonline.com/db776ee0-145c-47b0-a1b9-8a4da2dad75f/oauth2/v2.0/authorize?response_type=token&scope=https%3A%2F%2Fmicrosoft.onmicrosoft.com%2Fa9404161-811a-480e-a86c-082d92b9f4a2%2Fuser_impersonation%20openid%20profile&client_id=db7116e3-dbeb-4336-9525-07bd8dc6fede&redirect_uri=https%3A%2F%2Feccorevluumuipreproduatatm.trafficmanager.net%3A20200%2Fdashboard&state=13718827-145b-43cb-8cd6-31ffe506cf57&nonce=369dccb4-2543-4889-a1fe-f97424ae48c8&client_info=1&x-client-SKU=MSAL.JS&x-client-Ver=1.0.2&login_hint=user2%40uumtest1.net&login_req=96a19f2c-fb78-4ed8-93a2-7a63f1a0be69&domain_req=db776ee0-145c-47b0-a1b9-8a4da2dad75f&domain_hint=organizations&client-request-id=48d53b88-6190-4cd8-b3da-24bd2a99ac98&prompt=none



Error
https://eccorevluumuipreproduatatm.trafficmanager.net:20200/dashboard#error=consent_required&error_description=AADSTS65001%3a+The+user+or+administrator+has+not+consented+to+use+the+application+with+ID+%27db7116e3-dbeb-4336-9525-07bd8dc6fede%27+named+%27UUM-ConvergedApp%27.+Send+an+interactive+authorization+request+for+this+user+and+resource.%0d%0aTrace+ID%3a+0a051d3a-0c29-4696-a7ef-6b2e6d122400%0d%0aCorrelation+ID%3a+48d53b88-6190-4cd8-b3da-24bd2a99ac98%0d%0aTimestamp%3a+2019-07-01+03%3a11%3a52Z&state=13718827-145b-43cb-8cd6-31ffe506cf57

CorrelattionID      48d53b88-6190-4cd8-b3da-24bd2a99ac98
Timestamp-2019-07-01+03%3a11%3a52Z

**Comment by athreyee** [comment]
@sameerag We are approaching our Go live and couldn't proceed further because all the external tenants are unable to login to our systems because of this issue, can you please help on this to get us unblocked?

**Comment by sameerag** [comment]
@athreyee As we are following offline and this seems to a different issue than mentioned here, can we close this ticket?

**Comment by sameerag** [comment]
@athreyee I am closing this ticket. Please reopen in case you see a new issue.

**Comment by athreyee** [comment]
The issue got resolved by adding the client ID under Authorized client applications along with having in KnownClientApplications in the manifest file
